.github/workflows/R-CMD-check.yaml

```
@@ -56,10 +56,13 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::rcmdcheck
+          extra-packages: any::rcmdcheck, quarto=quarto-dev/quarto-r@libpaths
           needs: check
+          quarto-version: 1.8.15
 
       - uses: r-lib/actions/check-r-package@v2
+        env:
+          QUARTO_R_QUIET: FALSE
```

BTW the name has changed to `R_QUARTO_QUIET` when I did the last release 

```suggestion
          R_QUARTO_QUIET: FALSE
```